docs/INSTALLATION.md

```diff
@@ -174,24 +166,34 @@ that, you can override the `docker-compose.yml` configuration by adding a
 version: '3'
 services:
   app:
-    build:
-      context: .
-      dockerfile: Dockerfile
+    env:
+      - FETCH_SUBJECT=true
+      - CUSTOM_ENV=...
 ```
 
 Using `docker-compose up` automatically merges the override file in to the base configuration.
+For more about override files and merging configurations, see [https://docs.docker.com/compose/extends/](https://docs.docker.com/compose/extends/)
 
 ### Configuring a production environment
 
-The `docker-compose.yml` file provided is for a _development_ configuration;
-there are are a number of things you'll want to configure differently for
-production use, like setting the Rails application for production and setting
-up a reverse proxy such as Apache or Nginx to serve static assets. You can use the
-`docker-compose.yml` file as an example to write your own or simply override
-the existing configuration with `docker-compose.override.yml`. Both the
-override file and `docker-compose.production.yml` are gitignored.
+The `docker-compose.yml` file provided at the root of the project is for a
+_development_ configuration only. However, you can download the example
+`docker-compose.yml` file from [here](https://raw.githubusercontent.com/octobox/octobox/master/docs/examples/docker-compose.yml).
```

Comment:
This url link is no longer valid. I don't believe there is an example of a _production_ `docker-compose.yml` file anymore in this repo.